packages/nimble-components/src/anchor-button/tests/anchor-button.mdx

```diff
@@ -0,0 +1,26 @@
+import {
+    Controls,
+    DocsStory,
+    Meta,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import * as anchorButtonStories from './anchor-button.stories';
+
+<Meta of={anchorButtonStories} />
+
+<Title of={anchorButtonStories} />
+<Description of={anchorButtonStories} />
+
+<DocsStory of={anchorButtonStories.outlineAnchorButton} expanded={false} />
+<Controls of={anchorButtonStories.outlineAnchorButton} />
+<Stories of={anchorButtonStories} />
+
+# Usage Docs
+
+## Accessibility
```

Comment:
Interestingly, I just recently had to explain this behavior in [an SLE PR](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullRequest/528023#1690318766). Being able to point at these docs would have been great (I'd forgotten about accessibility.md).

What do you think about calling out the `title` guidance here too? I realize it's included in the API docs above and it's not necessarily part of accessibility, but I bet it would help people if we added something like "The text content should typically match the value provided for the `title` attribute."

Same for the other button components if you agree here.

Reply:
Discussed and this was referring to the content about title/tooltip for icon-only buttons. Currently that's captured in the "Icon Usage" section of the 'icon' API for buttons. Agreed that it makes sense to include that in this Accessibility section too.

Now the button accessibility text looks like the following:
<hr/>

When using the `nimble-button` with only an icon:
- Add the `content-hidden` attribute and provide text content within the button even though it will not be visible on the screen. The button will use the text content to configure the appropriate ARIA attributes internally to ensure the button is adequately accessible with a screen reader.
- Consider providing a tooltip to provide extra information (with the `title` attribute). Typically, the `title` attribute should match the text content in the button.

Reply:
> typically, the title attribute should match the text content in the button

Wait is that accurate advice? My uunderstanding is the title / tooltip should represent additional context while the aria label should describe the button.

icon: 📈
aria: buy stock
title / tooltip: buying stock increases the amount of stock you own

Reply:
I took out that line. I don't think it's always true. In some cases title will match (that's what we did for the table Collapse All button because we didn't think the icon would sufficiently convey that). But in other cases (well-known icon) it makes sense that the tooltip would be additional information. So it's probably case-by-case.